lambdas/preview/requirements.txt

```diff
@@ -0,0 +1,47 @@
+# TODO: add "../shared/" or "t4-lambda-shared" in a way that's not totally broken
```

Comment:
what's the broken part? does requirements.txt not support relative paths?

Reply:
Too much to fit into one comment :)

It supports relative paths, but only relative to the current working directory, not to the current file. If I use `-r ../shared/requirements.txt`, that will work relative to the current file - but not `../shared/`. Using `file:../shared/` should in theory work - but doesn't.

If I were to say `t4-lambda-shared==0.0.1`, that would check that the package is installed, though wouldn't be able to install it if it weren't. That would be fine... Except if I use `pip install -t dir ...`, as I do when building the zip file - in which case, even if the package is already installed, it still tries to install it again.

So I've given up on fighting it for now.

Reply:
k, a link to this comment would be nice `https://github.com/quiltdata/t4/pull/257#discussion_r259040374`